src/compas/geometry/shapes/capsule.py

```diff
@@ -263,6 +263,16 @@ def to_vertices_and_faces(self, u=10, v=10):
             nn = len(vertices) - 3 - (j + 1) % u
             faces.append([np, nn, nc])
 
+        if triangulated:
+            triangles = []
+            for face in faces:
+                if len(face) == 4:
```

Comment:
do we need to check for invalid cases, like `len(face) > 4` and raise?

Reply:
technically it is not possible to have an invalid case because we generate the faces and there are only tris and quads